aiidalab/app.py

```diff
@@ -330,7 +338,9 @@ def title(self):
     @property
     def url(self):
         """Provide explicit link to Git repository."""
-        return self._git_url
+        if self._registry_data is not None:
+            return self._registry_data.git_url
+        return None
```

Comment:
This could be done like this, but what you're doing might be clearer, i.e., more explicit?
```suggestion
        return getattr(self._registry_data, "git_url", None)
```

Reply:
Generally a fan of creating nice one-liners with `getattr`, however in this case this would not work, because `_registry_data` itself can be `None`, not `git_url`.

Reply:
But wouldn't `getattr` then return `None` as that's set to be the fallback/default value? Or would it completely fail due to an inconsistent type of `self._registry_data`? I'll just try this.

Reply:
Yup, in Python 3.7.5 I get that this will work fine, even if `self._registry_data` is `None`. It'll just return the third input parameter for `getattr`, i.e., `None`.

Reply:
TIL! :)